From cce4a6439a831622dc54d3d4fb840c4096aa8c62 Mon Sep 17 00:00:00 2001
From: Roy Franz <rfranz@marvell.com>
Date: Thu, 10 Jun 2021 14:54:58 -0700
Subject: [PATCH 1612/1921] soc: octeontx2-sdp: Fix formatting of SDP driver
 message

Fix formatting of SDP driver message that was missing a colon.

Signed-off-by: Roy Franz <rfranz@marvell.com>
Change-Id: I5b9d76dc034ed102976ee3104177a8ae17637976
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/54680
Reviewed-by: Roy Franz <Roy.Franz@cavium.com>
Reviewed-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/soc/marvell/octeontx2-sdp/sdp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/marvell/octeontx2-sdp/sdp.c b/drivers/soc/marvell/octeontx2-sdp/sdp.c
index 0f1dec3298e5..2924bf85c7b9 100644
--- a/drivers/soc/marvell/octeontx2-sdp/sdp.c
+++ b/drivers/soc/marvell/octeontx2-sdp/sdp.c
@@ -1069,7 +1069,7 @@ static void sdp_host_handshake_fn(struct work_struct *wrk)
 		neg_vf0_rings = host_rinfo.s.rppf;
 		neg_vfx_rings = host_rinfo.s.rpvf;
 		neg_vf = host_rinfo.s.numvf;
-		dev_info(&sdp->pdev->dev, "Negotiated VF0 rings:%d VFx rings%d VFs:%d\n",
+		dev_info(&sdp->pdev->dev, "Negotiated VF0 rings:%d VFx rings:%d VFs:%d\n",
 			 neg_vf0_rings, neg_vfx_rings, neg_vf);
 		host_rinfo.s.dir = FW_TO_HOST;
 		writeq(host_rinfo.u,
-- 
2.31.1

